__tests__/e2e/helpers/createForm.ts

```diff
@@ -553,30 +233,30 @@ const addAuthSettings = async (
     formSettings.authType === FormAuthType.SP
       ? ''
       : formSettings.authType === FormAuthType.SGID
-      ? ' App-only Login (Free)'
+      ? ' App-only Login'
       : formSettings.authType === FormAuthType.MyInfo
       ? ' with MyInfo'
       : ' (Corporate)'
   }`
 
   await page
     .locator('label', { has: page.getByRole('radio', { name }) })
-    .click()
+    .first() // Since 'Singpass' will match all radio options, pick the first matching one.
+    .click({ position: { x: 1, y: 1 } }) // Clicking the center of the sgid button launches the sgid contact form, put this here until we get rid of the link
```

Comment:
😆 how did you discover this worked?

Reply:
Haha, used debug mode which showed that it was clicking on the link in the middle of the button :')